pkg/pillar/cmd/volumemgr/volumemgr.go

```diff
@@ -495,6 +514,17 @@ func Run(ps *pubsub.PubSub) {
 	}
 }
 
+//handleInit if gc = true then this method will garbage collect all unused resource during init
+func handleInit(ctx *volumemgrContext, gc bool) {
```

Comment:
It would be more clear to have two separate functions since there is nothing in common between the true/false.

Could call one populateInitialBlobStatus() and the other GCUnusedBlobStatus or something like that.

Reply:
Changed.